shaders/glsl/renderchunk.vertex

```diff
@@ -240,18 +240,18 @@ if(isUnderWater > 0.9){
 	milkyFog.a = clamp(milkyFog.a, 0.0, 1.0);
 
 
-///// blended layer (mostly water) magic
+/// Blended layer (mostly water) magic
 #ifdef BLEND
-	//Mega hack: only things that become opaque are allowed to have vertex-driven transparency in the Blended layer...
-	//to fix this we'd need to find more space for a flag in the vertex format. color.a is the only unused part
+	// Mega hack: only things that become opaque are allowed to have vertex-driven transparency in the Blended layer...
+	// To fix this we'd need to find more space for a flag in the vertex format. color.a is the only unused part
 	bool shouldBecomeOpaqueInTheDistance = color.a < 0.95;
 	//bool shouldBecomeOpaqueInTheDistance = false;
 	if(shouldBecomeOpaqueInTheDistance) {
 		#ifdef FANCY  /////enhance water
 			float cameraDist = cameraDepth / FAR_CHUNKS_DISTANCE;
 			color = COLOR;
 		#else
-			//Completely insane, but if I don't have these two lines in here, the water doesn't render on a Nexus 6
+			// Completely stupid, but if I don't have these two lines in here, the water doesn't render on a Nexus 6

```

Comment:
It is original comment by mojang guy. Btw :-). It is not my

Reply:
Wait really lmao

Reply:
No need to fix. You can leave it as t is. I just noticed that it is original comment :-)